lib/ui/compositing/scene_builder.cc

```diff
@@ -226,6 +227,15 @@ void SceneBuilder::addPicture(double dx,
         SkPoint::Make(SafeNarrow(dx), SafeNarrow(dy)), picture->display_list(),
         !!(hints & 1), !!(hints & 2));
     AddLayer(std::move(layer));
+    return;
+  }
+
+  auto impeller_picture = picture->impeller_picture();
+  if (impeller_picture) {
+    auto layer = std::make_unique<flutter::AiksLayer>(
+        SkPoint::Make(SafeNarrow(dx), SafeNarrow(dy)),
+        std::move(impeller_picture), !!(hints & 1), !!(hints & 2));
```

Comment:
Lets just drop the hints since we don't use them for anything.

Reply:
Done